data/systemd/Makefile

```diff
@@ -53,8 +53,8 @@ clean:
 		sed s:@SNAP_MOUNT_DIR@:$(SNAP_MOUNT_DIR):g  | \
 		cat > $@
 
-# If SNAP_MOUNT_DIR uses non-default location then just copy the file.
-# This lets us reuse the sed rule above and get the right dependencies.
+# If SNAP_MOUNT_DIR uses non-default location then rename snap.mount
+# so that the directory name is encoded in the file name.
```

Comment:
The comment is a bit too easy to miss. Can't we always generate snap.mount and then copy it under the proper name in the `install` target?

Reply:
That would be a bit more verbose (filter out and separate install) but I think so, sure.

Reply:
Sorry, I completely misread the code.

Reply:
No worries, if you think that it should be that way I can tweak it further.